app/components/publishers/vacancies_component/vacancies_component.html.slim

```diff
@@ -0,0 +1,37 @@
+= render(Shared::DashboardComponent.new(heading: heading, navigation_items: vacancy_links, link: { url: organisation_jobs_path, text: t("buttons.create_job") } ))
+
+.vacancies-component
+  .govuk-grid-row.vacancies-component__header class="govuk-!-margin-top-7"
+    .govuk-grid-column-one-third
+      h2.govuk-heading-l class="govuk-!-margin-bottom-4" = t("jobs.#{@selected_type}_jobs_with_count_html", count: @vacancies.count)
```

Comment:
You might be able to use `==`. Cesi says

> Tip about slim, when you use the double == you do not need to have the translation key ending with _html

Conversation:

https://ukgovernmentdfe.slack.com/archives/CQ3UX00P6/p1610034056113300?thread_ts=1610033463.108900&cid=CQ3UX00P6

Reply:
yeah i saw this thread earlier and intended to align with this